### PR TITLE
Revert ":ship: Bump version v1.0.6 for kebechet (#460)"

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Revert kebechet to release fixes

### CODE REVIEW

File: kebechet/overlays/stage/imagestreamtag.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
     - name: latest
       from:
         kind: DockerImage
-        name: quay.io/thoth-station/kebechet-job:v1.0.6
+        name: quay.io/thoth-station/kebechet-job:v1.0.5
```
Comment (bissenbay) [inline kebechet/overlays/stage/imagestreamtag.yaml:11]: shouldn't it be `v0.6.0-dev`?

Comment (saisankargochhayat) [reply kebechet/overlays/stage/imagestreamtag.yaml:11]: https://quay.io/repository/thoth-station/kebechet-job?tab=tags
the 1.0.5 image seemed to be working

Comment (pacospace) [reply kebechet/overlays/stage/imagestreamtag.yaml:11]: wasn't the one that was not allowing release of investigator?

Comment (saisankargochhayat) [reply kebechet/overlays/stage/imagestreamtag.yaml:11]: Yes, if there are no commits, we will release source-management and kebechet and bump to 1.0.7

**Review by pacospace — APPROVED** [review]

**Pull request merged**